.github/workflows/push.yml

```diff
@@ -112,3 +112,26 @@ jobs:
                 extensions: intl
                 coverage: none # disable xdebug, pcov
           - run: composer global require friendsofphp/php-cs-fixer:2.14.* && ~/.composer/vendor/bin/php-cs-fixer fix --diff --dry-run
+
+  phpunit:
```

Comment:
Den neuen build brauchen wir, weil wir vorher phpunit nur in travis hatten?

Reply:
absolut, phpunit hatten wir bisher nur im travis.
wenn das hier funktioniert, würde ich den gleichen travis check entfernen.